woocommerce/woocommerce-gutenberg-products-block
# Product Gallery > Pager block: Fix block alignment

## What
This PR fixes a bug that was causing the Pager block to appear misaligned with the Large Image block.

Fixes #10781

## Why
As defined in the design for the block, the Pager block must appear below the Large Image block and centered with it.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. From your WordPress dashboard, go to Appearance > Themes. Make sure you have a block-based theme installed and activated. If not, you can install one from the Add New option. Block-based themes include "Twenty-twenty Two," "Twenty-twenty Three," etc.
2. On the left-hand side menu, click on Appearance > Editor > Templates
3. Find and select the 'Single Product' template from the list.
4. When the Classic Product Template renders, click on Transform into Blocks. This will transform the Classic template in a block template if you haven't done it before.
5. Inside the Page editor, click on the '+' button, usually found at the top left of the editing space or within the content area itself, to add a new block.
6. In the block library that pops up, search for the 'Product Gallery' block. Click on it to add the block to the template.
7. Check that the Pager block is displayed on the Editor.
8. Select the Product Gallery block and change the Align mode on the toolbar to "Full width"
![CleanShot 2023-08-30 at 14 23 39@2x](https://github.com/woocommerce/woocommerce-blocks/assets/20469356/a9b22546-714d-4e9d-9b76-f32ea4687fd8)
9. Make sure the block is correctly centered with the Large Image block.
10. Save the page and visit the product page, make sure the block is correctly being displayed and centered with the Large Image block.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
| <img width="1595" alt="CleanShot 2023-08-30 at 17 04 32@2x" src="https://github.com/woocommerce/woocommerce-blocks/assets/20469356/bed73658-7402-4e56-b88e-4f4926fa6456"> | <img width="1096" alt="CleanShot 2023-08-30 at 16 42 43@2x" src="https://github.com/woocommerce/woocommerce-blocks/assets/20469356/d776f315-4d40-452d-86f8-d253fa3872c5"> |

## WooCommerce Visibility


Required:
